packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -559,9 +559,7 @@ module.exports = function (log, config, bounces) {
         device: this._formatUserAgentInfo(message),
         date: date,
         email: message.email,
-        ip: message.ip,
         link: links.link,
-        location: message.location,
```

Comment:
Does this mean it's safe to not assign those two properties into `message` in the first place?  If it's to simple to make that change, I think we should.

Reply:
I think it's safe, I don't see where else we might have needed the ip and location from message (e.g., for security events or logs). I've gone ahead and remove these entirely from `message`.